{{cookiecutter.project_shortname}}/src/lib/components/{{cookiecutter.component_name}}.react.js

```diff
@@ -49,13 +49,13 @@ export default class {{cookiecutter.component_name}} extends Component {
     label: PropTypes.string.isRequired,
 
     /**
-     * The value displayed in the input
+     * The value displayed in the input.
      */
     value: PropTypes.string,
 
     /**
-     * Dash-assigned callback that should be called whenever any of the
-     * properties change
+     * Dash-assigned callback that should be called to exchange information
+     * related to property changes.
```

Comment:
hmm, I don't think that really tells component authors what they need to know about `setProps` - how about "to report property changes to Dash to make them available for callbacks."

Reply:
I agree it's not very descriptive; when I noticed that, slightly above, https://github.com/plotly/dash-component-boilerplate/blob/64406a8ecec85da56b40e930d48fa66d1a3b309c/%7B%7Bcookiecutter.project_shortname%7D%7D/src/lib/components/%7B%7Bcookiecutter.component_name%7D%7D.react.js#L23-L28 was telling component authors what they needed to know about `setProps`, I thought this comment should just not convey anything inaccurate (and remain short).

But your wording is definitely better, so I'll edit.